packages/app/src/app/pages/common/Modals/PreferencesModal/Badges/index.tsx

```diff
@@ -1,12 +1,20 @@
-import React from 'react';
-import { inject, observer } from 'app/componentConnectors';
+import React, {FunctionComponent} from 'react';
```

Comment:
```suggestion
import React from 'react';
```

For types we don't need to use named imports, we can just pull it off React